doc/cprover-manual/contracts-assigns.md

```diff
@@ -1,50 +1,144 @@
 [CPROVER Manual TOC](../../)
 
-# Assigns Clause
+# Assigns Clauses
```

Comment:
Having read to the end of this section, I really think this could do with an initial paragraph that sets the scene and explains why this even is useful or necessary.